api/WEBGL_compressed_texture_s3tc_srgb.json

```diff
@@ -5,10 +5,12 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/WEBGL_compressed_texture_s3tc_srgb",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "60",
+            "version_removed": "78"
```

Comment:
https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/renderer/modules/webgl/webgl_compressed_texture_s3tc_srgb.idl is still in Chromium, so I wonder if this is a testing issue? Perhaps it's hardware dependent and the hardware used changed?

Reply:
Just checked this more carefully, and it most likely is a hardware-dependent thing, yeah -- updating!